kvm-guests/common/lxc/iperf3.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# requires:
+#  bash
+#
+set -e
+set -o pipefail
+
+chroot_dir=${1}
+lxc_suffix=${2}
+file=iperf3-3.0.10-1.el6.x86_64.rpm
+
+if [ ! -f ${file} ]; then
+  chroot ${chroot_dir} /bin/bash -ex <<EOS
+    wget http://dl.fedoraproject.org/pub/epel/6/x86_64/${file}
+EOS
+fi
+
+chroot ${chroot_dir} /bin/bash -ex <<EOS
+  rpm -i --root=/var/lib/lxc/lxc${lxc_suffix}/rootfs --nodeps ${file}
```

Comment:
it doesn't work when using --installroot, it won't find the repository. I also tried with chroot, but the same propblem persisted, this was the only way I could find to install iperf3 on the LXC